src/applications/pensions/config/chapters/04-household-information/currentSpouseMaritalHistory.js

```diff
@@ -27,7 +29,7 @@ export default {
     type: 'object',
     required: ['currentSpouseMaritalHistory'],
     properties: {
-      currentSpouseMaritalHistory: radioSchema(Object.keys(radioOptions)),
+      currentSpouseMaritalHistory,
```

Comment:
Same as comment on checkboxGroupSchema change - I suggest holding off and doing these as a separate PR